utopia-remix/app/services/permissionsService.server.ts

```diff
@@ -39,6 +40,22 @@ export async function setProjectAccess(projectId: string, accessLevel: AccessLev
   await fgaService.updateAccessLevel(projectId, accessLevel)
 }
 
+export async function getAllPermissions(
+  projectId: string,
+  userId: string | null,
+  ownerId: string,
+): Promise<Record<fgaService.FgaUserProjectPermission, boolean>> {
+  const fgaPermissions = await fgaService.getAllPermissions(projectId, userId ?? ANONYMOUS_USER_ID)
+  // this is here since it's our internal logic, not part of FGA
+  if (userId === ownerId) {
```

Comment:
Seems we could avoid the `fgaService` call altogether then if the user is the owner, and instead use the same logic as `emptyUserPermissions` in that specific case